rs/bitcoin/kyt/tests/tests.rs

```diff
@@ -53,9 +53,17 @@ fn kyt_wasm() -> Vec<u8> {
 impl Setup {
     fn new(btc_network: BtcNetwork) -> Setup {
         let controller = PrincipalId::new_user_test_id(1).0;
-        let env = PocketIc::new();
+        // Enable nonmainnet_features to avoid CanisterInstallCodeRateLimited error
+        // for canister upgrades
+        let env = PocketIcBuilder::new()
+            .with_application_subnet()
+            .with_nonmainnet_features(true)
```

Comment:
Nit: It's a bit hard to read. How about `non_mainnet_features`?

Reply:
While I agree with the readability concern, this method is on PocketIc so I don't think we should change it as part of this MR (renaming the method would be a breaking change for PocketIC clients)

Reply:
What Greg said :-)